com.unity.multiplayer.mlapi/Tests/Runtime/MultiInstance/BaseMultiInstanceTest.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.Collections;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace MLAPI.RuntimeTests
+{
+    public class BaseMultiInstanceTest
```

Comment:
nit: could be abstract

Reply:
I'd call out @TwoTenPvP here since he might have some plans around this area.
looks like we keep adding more utilities, helpers, base classes and stuff into tests asmdef.
I wonder if we could outline the surface area and align on what are the "cool helpers/utils" we'd need and plans in our heads.
I'd like to avoid multiple people thinking about similar helpers/utils and coming up with very similar but still separate helpers/utils — maybe we could unify/consolidate some of the ideas + implementations together.

Reply:
also calling out @NoelStephensUnity here since he also had NetworkManagerHelpers implemented in the past.